vendor/vendor.json

```diff
@@ -983,14 +983,21 @@
 			"revisionTime": "2016-10-12T14:34:48Z"
 		},
 		{
+			"checksumSHA1": "ITwMGcU7ZNb0nmOHcKXMjK2i69g=",
+			"path": "google.golang.org/api/compute/v0.beta",
+			"revision": "e665075b5ff79143ba49c58fab02df9dc122afd5",
```

Comment:
It looks like you're adding a whole new entry rather than overwriting the API.  If we're going to do this we ought  not to leave around vestigial code grabbing v1.0